test/integration/Test/Integration/Framework/DSL.hs

```diff
@@ -8,34 +14,80 @@ module Test.Integration.Framework.DSL
     -- * Expectations
     , expectSuccess
     , expectError
+    , expectFieldEqual
+    , expectFieldNotEqual
     , expectResponseCode
     , Headers(..)
     , Payload(..)
     , RequestException(..)
 
+    -- * Lens
+    , addressPoolGap
+    , balanceAvailable
+    , balanceTotal
+    , delegation
+    , passphraseLastUpdate
+    , walletId
+    , walletName
+    , state
+
     -- * Helpers
     , (</>)
     , (!!)
+    , getFromResponse
     , json
     ) where
 
 import Prelude hiding
     ( fail )
 
+import Cardano.Wallet.Api.Types
+    ( ApiT (..) )
+import Cardano.Wallet.Primitive.AddressDiscovery
+    ( AddressPoolGap, getAddressPoolGap, mkAddressPoolGap )
+import Cardano.Wallet.Primitive.Types
+    ( PoolId (..)
+    , WalletBalance (..)
+    , WalletDelegation (..)
+    , WalletId (..)
+    , WalletName (..)
+    , WalletPassphraseInfo (..)
+    , WalletState (..)
+    )
+import Control.Lens
+    ( set, view )
```

Comment:
If `set` and `view` are all you need, I'd suggest to pull them from: https://hackage.haskell.org/package/generic-lens-1.1.0.0/docs/Data-Generics-Internal-VL-Lens.html and avoid the extra `lens` dependency for the whole package (which is kinda big, trying to get rid of this on the main library too...)